NEWS.md

```diff
@@ -415,6 +415,8 @@ Library improvements
     * `mktemp` and `mktempdir` now take an optional argument to set which
       directory the temporary file or directory is created in.
 
+    * New, generational garbage collector ([#5227]). In some use cases it can reduce the worst case time for garbage collection by a factor of ten. Julia can now be used for soft real-time applications like audio processing or high-level robot control.
```

Comment:
This claim is based on the following test script:
https://gist.github.com/ufechner7/39124402f17eee3f40a8

First of all I want to say, that the new garbage collector performs MUCH better than the old one.
With Julia 0.4.0-dev+6909 I achieve a jitter of about 2ms and a crest factor of the execution time of my workload of about 1.23. (Core i7, 3.4 GHz, sandybridge, no hyperthreading, Ubuntu 12.04).
With Julia 0.3.10 the jitter is about 20 ms and the crest factor about 2.5. This means, for the given example Julia 0.4 is by a factor of 10 better then Julia 0.3. This is already very nice.